### PR TITLE
refactor: remove reactivity transform

### LINKED ISSUE #57: Reactivity transform
As reactivity transform is planned to be removed in the future, should we migrate off of it soon or keep it up for a little longer?

### PR DESCRIPTION
**Pull request by manniL**
Resolves #57

### CODE REVIEW

File: client/components/PluginItem.vue [hunk 1]
```diff
@@ -6,15 +6,16 @@ const props = defineProps<{
...
-const normalized = $computed((): NuxtPlugin => {
+const normalized = computed((): NuxtPlugin => {
   if (typeof props.plugin === 'string')
     return { src: props.plugin }
   return props.plugin
 })
 
-const shortPath = $computed(() => config ? getShortPath(normalized.src, config.rootDir, true)! : '')
+// TODO: Check if correct
+const shortPath = computed(() => config ? getShortPath(normalized.value.src, config.value?.rootDir ?? '', true)! : '')
```
Comment (manniL) [inline client/components/PluginItem.vue:18]: What would be preferred here? The current code or an exists assertion?

```suggestion
const shortPath = computed(() => config ? getShortPath(normalized.value.src, config.value!.rootDir, true)! : '')
```

File: client/components/PluginItem.vue [hunk 2]
```diff
@@ -6,15 +6,16 @@ const props = defineProps<{
...
 
-const normalized = $computed((): NuxtPlugin => {
+const normalized = computed((): NuxtPlugin => {
   if (typeof props.plugin === 'string')
     return { src: props.plugin }
   return props.plugin
 })
 
-const shortPath = $computed(() => config ? getShortPath(normalized.src, config.rootDir, true)! : '')
+// TODO: Check if correct
```
Comment (manniL) [inline client/components/PluginItem.vue:17]:
```suggestion
```

Will be removed after discussion of L18 has been resolved

File: client/components/PluginItem.vue (hunk 1 above)
Comment (antfu) [reply client/components/PluginItem.vue:18]: It should check `config.value ?`

**Comment by antfu** [comment]
I would miss it

**Comment by manniL** [comment]
> I will miss it :P

Me too 🙈 
But I totally understand why it will be gone in a bit...

**Pull request merged**